dev/com.ibm.ws.logging.osgi/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -14,76 +14,73 @@
 
 trace.format=Trace format
-trace.format.desc= This format is used for the trace log.
+trace.format.desc= This format is used for the trace log. Avoid trouble: Use the com.ibm.ws.logging.trace.format property, except in cases where you are trying to change the configuration dynamically after startup.
 trace.basic=Use the basic trace format.
 trace.enhanced=Use the enhanced basic trace format.
 trace.advanced=Use the advanced trace format.
 
 isoDateFormat=Use the ISO 8601 date format
-isoDateFormat.desc=If this attribute is set to true, log entries in the messages.log, trace.log, and FFDC log files will use the ISO 8601 date and time format. The default value is false.
+isoDateFormat.desc=The date and time use a locale-specific format or the ISO-8601 format. You can specify true or false for the value of the attribute or the value of the equivalent property. The default value is false. Avoid trouble: Use the com.ibm.ws.logging.isoDateFormat property, except in cases where you are trying to change the configuration dynamically after startup. \
+If you specify a value of true, the ISO-8601 format is used in the messages.log file, the trace.log file, and the FFDC logs. The format is yyyy-MM-dd'T'HH:mm:ss.SSSZ. \
+If you specify a value of false, the date and time are formatted according to the default locale set in the system. If the default locale is not found, the format is dd/MMM/yyyy HH:mm:ss:SSS z.
 
 filter.sensitive=Suppress sensitive trace
 filter.sensitive.desc=The server trace can expose sensitive data when tracing untyped data, such as bytes received over a network connection. \
-If true, prevent potentially sensitive information from being exposed in log and trace files. The default value is false.
+If true, prevent potentially sensitive information from being exposed in log and trace files. The default value is false. Avoid trouble: Use the com.ibm.ws.logging.filter.sensitive property, except in cases where you are trying to change the configuration dynamically after startup.
 
 hide.messages=Messages to be hidden
-hide.messages.desc=The list of messages, separated by a comma, that are configured to be hidden from the console.log and message.log files. If the messages are configured to be hidden, then they are redirected to the trace.log file.
+hide.messages.desc=The list of messages, separated by a comma, that are configured to be hidden from the console.log and message.log files. If the messages are configured to be hidden, then they are redirected to the trace.log file. Avoid trouble: Use the com.ibm.ws.logging.hide.messages property, except in cases where you are trying to change the configuration dynamically after startup.
```

Comment:
Change
"The list of messages, separated by a comma, that are configured to be hidden from the console.log and message.log files."
to
"The list of messages, which are separated by a comma, that are configured to be hidden from the console.log and message.log files. "